### PR TITLE
[Rev 1 Demo] Add mailto and tel links

### LINKED ISSUE #638: [Rev 1 Demo] Add mailto link for shop email and tel for phone number in shop profiles

### PR DESCRIPTION
**Pull request by LeonSo7**
[Rev 1 Demo] Add mailto and tel links

Close #638

### CODE REVIEW

File: src/pages/shop/[id]/index.tsx [hunk 1]
```diff
@@ -220,11 +220,13 @@ const Profile: NextPage = () => {
             <br></br>
             <span className={styles.shopProfileSubHeaderText}>
               <i className="pi pi-phone"></i>
-              {` ${shop.phoneNumber}`}
+              <a
+                href={`tel: ${shop.phoneNumber.replace(/\D/g, "")}`}
+              >{` ${shop.phoneNumber}`}</a>
```
Comment (arkinmodi) [inline src/pages/shop/[id]/index.tsx:225]: To be complete, we should probably add the country code. We said in the SRS that we are assuming the shop is in Canada, so you can hard code `+1` to this.

File: src/pages/shop/[id]/index.tsx [hunk 2]
```diff
@@ -220,11 +220,13 @@ const Profile: NextPage = () => {
...
             </span>
             <span className={styles.shopProfileSubHeaderText}>
               <i className="pi pi-envelope"></i>
-              {` ${shop.email}`}
+              <a href={`mailto: ${shop.email}`}>{` ${shop.email}`}</a>
```
Comment (arkinmodi) [inline src/pages/shop/[id]/index.tsx]: Looking at the examples, I don't think there is normally a space.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a

Comment (LeonSo7) [reply src/pages/shop/[id]/index.tsx]: Works with or without the link. Lots of examples with a space.
https://www.tutorialspoint.com/html/html_email_links.htm#

I removed it anyways

File: src/pages/shop/[id]/index.tsx (hunk 1 above)
Comment (LeonSo7) [reply src/pages/shop/[id]/index.tsx:225]: Sure, added +1

**Review by arkinmodi — APPROVED** [review]

**Pull request merged**